F1/F1/Constructor/ConstructorViewModel.swift

```diff
@@ -18,22 +19,26 @@ class ConstructorViewModel {
         self.repository = repository
         self.delegate = delegate
     }
-    var resultsCount: Int {
-        return constructorResults?.count ?? 0
-    }
 
-    func result(atIndex: Int) -> Race? {
-        if constructorResults?.count ?? 0 > atIndex {
-            return constructorResults?[atIndex] ?? nil }
-        return nil
+    // MARK: Computed Variables
+    var resultsCount: Int {
+        constructorResults?.count ?? 0
     }
 
     var drivers: String {
-        return "\(constructorResults?.first?.results.first?.driver.code ?? "") | \(constructorResults?.first?.results.last?.driver.code ?? "")"
+        "\(constructorResults?.first?.results.first?.driver.code ?? "") | \(constructorResults?.first?.results.last?.driver.code ?? "")"
     }
 
+    // MARK: Functions
+    func result(atIndex: Int) -> Race? {
+        if constructorResults?.count ?? 0 > atIndex, let result = constructorResults {
```

Comment:
func result(atIndex: Int) -> Race? {
   constructorResults?.count ?? 0 > atIndex ? constructorResults?[atIndex] : nil
}

removes a lot of lines